Views/CropsView.swift

```diff
@@ -324,14 +292,13 @@ class CropsView: UIView, UITableViewDataSource, UITableViewDelegate {
     }
 
     let managedContext = appDelegate.persistentContainer.viewContext
-    let cropsEntity = NSEntityDescription.entity(forEntityName: "Crops", in: managedContext)!
-    let crop = NSManagedObject(entity: cropsEntity, insertInto: managedContext)
+    let crop = Crops(context: managedContext)
     let plot = fetchPlot(withName: plotName)
 
-    crop.setValue(plot, forKey: "plots")
-    crop.setValue(name, forKey: "name")
-    crop.setValue(surfaceArea, forKey: "surfaceArea")
-    crop.setValue(startDate, forKey: "startDate")
+    crop.plots = plot
```

Comment:
Crop.plot**s** ? Why can a crop get several plots? And if it can, why is it set to only one plot (and not at least a collection of plots (`= [plot]`) ?

Reply:
The name of the relationship is plural but it's one-to-many relationship. I will change the name to singular